client/src/modules/stock/entry/modals/lots.modal.js

```diff
@@ -336,7 +342,7 @@ function StockDefineLotsModalController(
           }
         });
       }
-      return Promise.all(promises)
+      Promise.all(promises)
```

Comment:
You might need to return this for the loading indicator to work, if there is a loading indicator on this modal.  If there isn't, we really should have one.